www/templates/user-vhost-warning

```diff
@@ -0,0 +1,22 @@
+Dear {group}:
+
+The OCF staff noticed that your group's website [1] does not have either the "Hosted
+by OCF" banner [2] or University disclaimer [3] required by OCF and university virtual
+hosting rules. All student group websites are required to display the "Hosted by OCF"
+banner as well as the student group disclaimer. Unfortunately, if you do not add both
+of them, we may be forced to temporarily disable your website to comply with the OCF's
+and university's requirements. Please add both sometime within the next 2 weeks to
+prevent disabling of your website.
+
+Your username is "{username}".
+
+If you've forgotten how to login, instructions can be found here [4]. If you've
+forgotten your password, you can reset it here [5].
+
+Feel free to reply all to this email if you have any questions or if we can help with making the change! Please include help@ocf.berkeley.edu if you would like a response.
```

Comment:
If you use an RT ticket in the help queue, then `help@ocf.berkeley.edu` will already be the reply-to address. There wouldn't be a difference between reply and reply all either.